examples/config.kinesis.minimal.hocon

```diff
@@ -0,0 +1,30 @@
+# The minimum required config options for loading from kinesis
+"input": {
+  "type": "kinesis"
+  "streamName": "test-kinesis-stream"
```

Comment:
Isn't `appName` also required for minimal app?

Reply:
It has [default value](https://github.com/snowplow/snowplow-elasticsearch-loader/pull/203/files#diff-3591236f1f734d63ce617d707d8d2d7f6e25d637d28074ed1543427d496d695dR4).

Reply:
@istreeter do you think it should be a default for other apps?